### ISSUE TITLE
Reading in isoforms and SNVs from UniProt XML databases

### ISSUE THREAD

**Issue by rmillikin** [smith-chem-wisc/MetaMorpheus#1842]
Apparently we're not reading these in, though we do read in the protein's annotated modified residues. It may or may not be a good idea to read in all SNVs, etc. but maybe we should provide options to search these annotated species.

Counting entries in a human canonical reviewed XML:

Entries: 20397 proteins
"isoform" XML tags: 32925 (though it seems that perhaps this includes the canonical sequences?)
"modified residue" tags: 52891
"sequence variant" tags: 79669

**Comment by acesnik** [comment]
I agree this could be interesting functionality. However, I believe UniProt's sequence variant tags won't be very useful for searches because they're from all sorts of samples; Gloria Sheynkman showed something similar with dbGaP in her 2014 article.

For isoforms, one can search the FASTA canonical & isoform database and use GPTMD to get modifications.
